x-pack/plugins/fleet/public/applications/integrations/app.tsx

```diff
@@ -60,6 +63,7 @@ export const IntegrationsAppContext: React.FC<{
     theme$,
   }) => {
     const isDarkMode = useObservable<boolean>(startServices.uiSettings.get$('theme:darkMode'));
+    const CloudContext = startServices.cloud?.CloudContextProvider || EmptyContext;
```

Comment:
what happens with on-prem installations? is the chat component visible there?

Reply:
No.  Not unless they decide to set the config for chat, _and_ have their endpoint send the drift onReady message.